### PR TITLE
Registration page updates

### PR DESCRIPTION
**Pull request by DanHillesheim**
hot fix for cnet campaign - Need to review with marketing team before merge.

## Type of change

- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

Fix existing registration pages and add new variants for CNET campaign

## Code changes

register.component.html - minor class additions and markup editing
pages.scss - update/add some styles
4(four) svg files - new logo files

## Testing requirements

These URLs should be tested for accuracy and UI consistency:
/#/register?premium=purchase&layout=cnetcmpgnind
/#/register?org=teams&layout=cnetcmpgnteams
/#/register?org=enterprise&layout=cnetcmpgnent

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Comment by Hinton** [comment]
We need a better way to handle these variants of the register page.

**Comment by DanHillesheim** [comment]
Agreed. Not the cleanest, but we'll work toward something better soon.

**Comment by Hinton** [comment]
Layout enterprise2 is also broken now.

**Comment by DanHillesheim** [comment]
> Layout enterprise2 is also broken now, and teams1 looks a bit off.

I will look at the history, but yes, it's currently like that on the live vault page. I believe it has been like that for a bit.

File: src/app/accounts/register.component.html
Comment (Hinton) [inline src/app/accounts/register.component.html:30]: This seems to be duplicated for all layouts?

Comment (DanHillesheim) [reply src/app/accounts/register.component.html:30]: Yes. It is duplicated. There is a request for differentiation between them but a final decision hasn't been made. I'll try again to get feedback. Until then, are you suggesting we eliminate the dupes until variations are necessary? I'm fine leaving them until that determination is made, but will defer to your preference.

Comment (Hinton) [reply src/app/accounts/register.component.html:30]: We can postpone it, but we shouldn't merge it before it's resolved.

Comment (DanHillesheim) [reply src/app/accounts/register.component.html:30]: We need to leave the dupes. Freshsales triggers our drip campaigns based on URL params. Unless there's a better way to accomplish this, we'll have to roll with this. We would like to get this resolved as soon as we can so we can make sure the campaign gets tracked properly. Let me know if there is anything else you can see that needs to be addressed. Thanks!

Comment (Hinton) [reply src/app/accounts/register.component.html:30]: Can't we have `<header class="header" *ngIf="layout === 'teams1' || layout === 'enterprise1'">`?

Comment (DanHillesheim) [reply src/app/accounts/register.component.html:30]: I'm not sure we can default to teams1 if layout != enterprise1. I'll ask the demand gen team if that's acceptable.

**Comment by Hinton** [comment]
Everything except `layout=teams2` looks good. We should look into breaking this component up, in the future.

**Comment by DanHillesheim** [comment]
teams2 was left out of the body, I added it back in.

**Review by Hinton — APPROVED** [review]

**Review by Hinton — APPROVED** [review]

**Pull request merged**
